.github/workflows/publish-beta-release.yml

```diff
@@ -0,0 +1,74 @@
+name: Publish Beta Release to Github Only
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  save_artifacts:
+    name: 'Build VSIX Files'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.13.x'
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.ref }}
+      - name: Create and Package Extensions
+        run: |
+          npm install -g lerna
+          npm install
+          npm run compile
+          npm run vscode:package
+      - name: Stage Artifacts
+        run: |
+          mkdir extensions
+          find packages -name "*.vsix" -type f -exec cp {} ./extensions \;
+          cp ./packages/salesforcedx-vscode/CHANGELOG.md ./extensions
+      - name: Upload Extensions
+        uses: actions/upload-artifact@v3
+        with:
+          name: VS Code Extensions
```

Comment:
nit: should we include the tag here or is there no shared view of all uploads across actions?

Reply:
The artifact is limited to the workflow run, so we don't need to include the tag (and we save a few seconds if we don't wait for it)